vendor/modules.txt

```diff
@@ -2461,10 +2467,10 @@ k8s.io/metrics/pkg/client/custom_metrics/fake
 k8s.io/metrics/pkg/client/custom_metrics/scheme
 k8s.io/metrics/pkg/client/external_metrics
 k8s.io/metrics/pkg/client/external_metrics/fake
-# k8s.io/mount-utils v0.0.0 => ./staging/src/k8s.io/mount-utils
+# k8s.io/mount-utils v0.20.0-alpha.3 => k8s.io/mount-utils v0.20.0-alpha.3
 ## explicit
 k8s.io/mount-utils
-# k8s.io/mount-utils => ./staging/src/k8s.io/mount-utils
+# k8s.io/mount-utils => k8s.io/mount-utils v0.20.0-alpha.3
```

Comment:
hmm are you running ?

```
hack/pin-dependency.sh github.com/docker/docker bd33bbf0497b2327516dc799a5e541b720822a4c
hack/update-vendor.sh
```
to do the updates?

Reply:
yeah!

Reply:
&& 
hack/pin-dependency.sh github.com/containerd/containerd v1.4.1
hack/pin-dependency.sh github.com/google/cadvisor b31c0ce7a97f1099fd2c94d6bd4d01e3795ceaa5
hack/update-vendor.sh